tests-dev/ufs_test.yaml

```diff
@@ -0,0 +1,583 @@
+s2swa_32bit_intel:
```

Comment:
I do not see in this file which tests are 'baseline' tests. For example, when I run `./ufs_test.sh -r -s -c` which tests will be run in order to create new baselines?

Reply:
I would rather use dependency key with test cases. Script will let dependency case run first.

Reply:
I am sorry, but I do not understand this. Dependency has nothing to do with baselines. Those are two completely different concepts.

Reply:
There are tests (control_p8_mynn for example) that do not depend on any other tests, and there are no tests that depend on this test. Yet this test (control_p8_mynn) needs baselines.

Reply:
control_p8_mynn is treated as independent baseline ok in the test. runtime options of -l -n -m -c -b should work as well.

Reply:
If I run `./ufs_test.sh -r -s -c` which tests will be executed?
Will `cpld_control_p8` test be executed?
Will `cpld_2threads_p8` test be executed?

Reply:
only cpld_control_p8 will be created -c.

Reply:
I'm not sure I understand what you mean by 'only cpld_control_p8 will be created -c'. 

My question was whether both `cpld_control_p8` and `cpld_2threads_p8` tests will be executed?

Reply:
cpld_2threads_p8 will not be executed with -c

Reply:
Where and how is that specified?

Reply:
When I run `./ufs_test.sh -r -s -c` (on Hera) and look at generated workflow file I see both `cpld_control_p8` and `cpld_2threads_p8` listed:

```
$ egrep 'cpld_control_p8_intel|cpld_2threads_p8_intel' rocoto_workflow.xml
    <task name="cpld_control_p8_intel" maxtries="3">
      <command>bash -c 'set -xe -o pipefail ; &PATHRT;/run_test.sh &PATHRT; &RUNDIR_ROOT; cpld_control_p8 cpld_control_p8_intel s2swa_intel 2>&amp;1 | tee &LOG;/run_cpld_control_p8_intel.log' </command>
      <jobname>cpld_control_p8_intel</jobname>
      <join>&RUNDIR_ROOT;/cpld_control_p8_intel.log</join>
      <dependency> <and> <taskdep task="compile_s2swa_intel"/> <taskdep task="cpld_control_p8_intel"/> </and> </dependency>
    <task name="cpld_2threads_p8_intel" maxtries="3">
      <command>bash -c 'set -xe -o pipefail ; &PATHRT;/run_test.sh &PATHRT; &RUNDIR_ROOT; cpld_2threads_p8 cpld_2threads_p8_intel s2swa_intel 2>&amp;1 | tee &LOG;/run_cpld_2threads_p8_intel.log' </command>
      <jobname>cpld_2threads_p8_intel</jobname>
      <join>&RUNDIR_ROOT;/cpld_2threads_p8_intel.log</join>
```
but you said that cpld_2threads_p8 will not be executed. How did you check that only cpld_control_p8 will be executed? Where is that specified? Based on this workflow file both will be executed.